src/views/auth/SignUp.vue

```diff
@@ -119,12 +110,18 @@ export default {
             confirm: this.form.confirm
           }
 
-          this.$store.dispatch('signup', {
+          this.signup({
             email: formData.email,
             password: formData.password
           })

```

Comment:
Where is `then` block to handle $router redirect to the `/home` page after signup?